tests/interactive/test_app.py

```diff
@@ -21,8 +21,10 @@ async def test_inputs():
         app = cast(InputApp, pilot.app)
 
         # Test no input
-        assert app.output_text_area.text == "No input"
-        assert app.current_module is None
+        assert (
+            app.output_text_area.text
+            == "(Span[0:1](text=''), 'Could not parse entire input!')"
+        )
```

Comment:
this behaviour change is unexpected. It feels like it would be good to avoid displaying an error first thing when a user opens the app